bucket/xmind.json

```diff
@@ -8,14 +8,14 @@
     },
     "architecture": {
         "64bit": {
-            "url": "https://dl3.xmind.net/XMind-for-Windows-64bit-11.1.2-202111071931.exe#/dl.7z",
+            "url": "https://dl3.xmind.net//XMind-for-Windows-64bit-11.1.2-202111071931.exe#/dl.7z",
```

Comment:
Why use 2 slashes? This URL now results in 403 error. Same for autoupdate. You can remove the beginning slash in checker script by `.TrimStart('/')`

Reply:
Apologize for my carelessness